pages/events/[...slug].vue

```diff
@@ -95,7 +94,7 @@ useHead({
           <ul v-else-if="sectionId === 'speakers'">
             <li v-for="speakerId in sectionContent.value" :key="speakerId">
               <!-- <a :href="`/speakers/${speakerId}`">{{ `${speakers[speakerId].name} - ${speakers[speakerId].company}` }}</a> -->
-              <div>{{ `${speakers[speakerId].name} - ${speakers[speakerId].company}` }}</div>
+              <div>{{ `${getSpeakerName(speakerId)} - ${getSpeakerCompanyName(speakerId)}` }}</div>
```

Comment:
Please refactor this, as using multiple functions to search in the same array is less optimal than finding the event once and passing it as an object.

Reply:
the json restructure fixed it